binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -241,7 +242,9 @@ public void initialize() {
             }
             onUpdate();
         } else {
-            logger.warn("Cannot connect to hue bridge. IP address or user name not set.");
+            String warn = "Cannot connect to hue bridge. IP address or user name not set.";
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, warn);
+            logger.warn(warn);
```

Comment:
> this message needs at least logged (by the updateStatus)

In general I agree, but see
 See https://github.com/eclipse/smarthome/pull/249/files#diff-24d71ac793f6de6c8557ea06292497ddL134
which means that we will very soon also add events for this - and then the better place to log might be an event listener and this is already in place for the item events.

> with at least warn severity, because it is an error.

This probably leads to a much broader discussion: What is an error and how to use severities in general? It is no system error and as such an expected behavior - so I would rather only use debug logging. You do not know if it is fully ok for a device to go OFFLINE and the user expects it to (because he powered it off). So flooding the log with errors is not a good idea. What you expect are rather notifications to the user and I doubt that the log is the best place for this. I'd suggest to rather follow up on this discussion for such use cases: https://groups.google.com/d/msg/openhab/AoJW4b7uiqY/S761KUXIR_4J